src/handlers/payout/action.ts

```diff
@@ -87,9 +87,9 @@ export const handleIssueClosed = async () => {
   }
 
   logger.info(`Handling issues.closed event, issue: ${issue.number}`);
-  if (!autoPayMode) {
-    logger.info(`Skipping to generate permit2 url, reason: { autoPayMode: ${autoPayMode}}`);
-    return `Permit generation skipped since autoPayMode is disabled`;
+  if (!paymentPermitMaxPrice) {
```

Comment:
```suggestion
  if (paymentPermitMaxPrice == 0) {
```

Suggesting above to avoid confustion of `paymentPermitMaxPrice` type.